lib/smart_answer/calculators/child_maintenance_calculator.rb

```diff
@@ -173,5 +170,39 @@ def total_yearly_fees(collect_fees)
     def self.child_maintenance_data
       @child_maintenance_data ||= YAML.load_file(Rails.root.join("lib/data/child_maintenance_data.yml"))
     end
+
+    def state_benefits?
+      benefits.present? && benefits.is_a?(Array) && valid_state_benefits?
```

Comment:
As I went through the possible unit test edge cases especially with the an empty or invalid string getting passed to benefits I believe that is where I added the `.present?`. At this point happy to remove `.present?`